src/ripple/app/ledger/LedgerHistory.cpp

```diff
@@ -384,7 +391,7 @@ void LedgerHistory::handleMismatch (
         log_one (validLedger, (*v)->key(), "built", j_);
 }
 
-void LedgerHistory::builtLedger (Ledger::ref ledger)
+void LedgerHistory::builtLedger (Ledger::ref ledger, Json::Value&& consensus)
 {
```

Comment:
Why force pass by rvalue ref? Passing by value cost one extra move constructor when the caller uses an rvalue and the cost `Json::Value`'s rvalue constructor is low. The advantage of passing by value is callers can use both rvalues and lvalues as parameters.